roles/forklift_versions/library/forklift_versions.py

```diff
@@ -87,6 +87,8 @@ def main():
                         'pulp_repositories_version': version['pulp'],
                         'puppet_repositories_version': version['puppet'],
                         }
+                if 'pulpcore' in version:
```

Comment:
this is separate, so the var is undefined for versions that don't use the split pulpcore